src/main/features/dashboard/routes/directions-questionnaire.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from 'express'
+
+import { Paths } from 'dashboard/paths'
+import { ErrorHandling } from 'shared/errorHandling'
+import { ClaimStoreClient } from 'claims/claimStoreClient'
+import { Claim } from 'claims/models/claim'
+import { User } from 'idam/user'
+
+const page = Paths.directionsQuestionnairePage
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(
+    page.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      const { externalId } = req.params
+      const user: User = res.locals.user
+      const claim: Claim = await new ClaimStoreClient().retrieveByExternalId(externalId, user)
+
+      res.render(page.associatedView, {
+        deadline: claim.respondedAt.clone().add(19, 'days'),
```

Comment:
you can do a filter in this view for this now ` | addDays(30)`
see:
https://github.com/hmcts/cmc-citizen-frontend/pull/720

Reply:
I would advise against doing it in template as this is business logic. Andrew's solution of mutating dates in template is perfectly fine for examples though.

Reply:
sure, I think its also, 14 days + 5 days service, if you want to separate it out?

Reply:
I have left it here as it was. I used `addDays` in template: src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk:81
src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk:61